components/dashboard/src/data/git-providers/unified-repositories-search-query.ts

```diff
@@ -26,9 +35,34 @@ export const useUnifiedRepositorySearch = ({
     onlyConfigurations = false,
     showExamples = false,
 }: UnifiedRepositorySearchArgs) => {
-    const suggestedQuery = useSuggestedRepositories({ excludeConfigurations });
+    // 1st data source: suggested SCM repos + up to 100 imported repos.
+    // todo(ft): look into deduplicating and merging these on the server
+    const suggestedQuery = useSuggestedRepositories({ excludeConfigurations: true });
```

Comment:
It's weired to do this change, I image that `suggestedRespositories` has some logic inside like base on users' activity recommend them with different priority of items.

In current implement, we will filter out configurations and append only via name searching.

Just feel it's weired, not blocking

Reply:
It leads a condition: I use `bitbucket.gitpod-dev.com/scm/tes/ten-k-1304` to start a workspace, and back to dashboard, the first listed context url in new workspace page is not `1304`

Reply:
That's fair, we should not make this reorder all projects all of a sudden. Reverting this change

Reply:
[`bbd8559` (#20151)](https://github.com/gitpod-io/gitpod/pull/20151/commits/bbd8559beae585c89f180dd80025a04d65133d75)